mycroft/skills/phillips_hue/vocab/en-us/IncreaseBrightnessKeyword.voc

```diff
@@ -0,0 +1,4 @@
+increase the brightness of my lights
+increase the brightness of the lights
+turn up my lights
+turn up the lights
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `IncreaseBrightnessKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.